proto-compiler/README.md

```diff
@@ -0,0 +1,7 @@
+## How to compile fresh proto structs
+
+* `git clone https://github.com/tendermint/tendermint` into the repository `target/` folder.
+* `cargo run` in the compiler folder.
+
+The resultant structs will be created in the `tendermint-proto/src/prost` folder.
```

Comment:
Wasn't the folder renamed from tendermint-proto to proto? Would be good to double check if that still works.

Reply:
Yes, good point. Opened a quick issue about it.